tests/test_writer.py

```diff
@@ -173,10 +174,11 @@ def test_iter_lines__no_warn_if_only_unhashable_packages(writer, from_line):
     lines = writer._iter_lines(ireqs, hashes=hashes)
 
     expected_lines = (
-        "file:///unhashable-pkg1/#egg=unhashable-pkg1",
-        "file:///unhashable-pkg2/#egg=unhashable-pkg2",
+        r"file:///([A-Z]:/)?unhashable-pkg1#egg=unhashable-pkg1$",
+        r"file:///([A-Z]:/)?unhashable-pkg2#egg=unhashable-pkg2$",
     )
-    assert tuple(lines) == expected_lines
+    for line, pattern in zip(lines, expected_lines):
+        assert re.match(pattern, line)
```

Comment:
`pip install re-assert` could be used for asserting regexps. It shows where regex match assertion failed, which is pretty helpful.

Reply:
That looks neat! Possibly overkill for this case, but either way, I'd rather not saddle this PR with an extra dev dependency. If you'd like to use `re-assert`, could it be in a separate PR?

Reply:
Yeah, of course 👍🏼